### PR TITLE
Update Russian translation of Plug lesson to version 1.1.1

### PR DESCRIPTION
**Pull request by martosaur**
As of https://github.com/elixirschool/elixirschool/issues/1043
/cc @nscyclone

### CODE REVIEW

File: ru/lessons/specifics/plug.md [hunk 1]
```diff
@@ -1,54 +1,171 @@
 ---
-version: 0.9.0
+version: 1.1.1
 title: Plug
 ---
 
-Если вы знакомы с `Ruby`, то можете думать о `Plug` как о комбинации `Rack` и `Sinatra`. Во-первых, `Plug` это набор договорённостей и спецификаций, позволяющих создавать универсальные компонуемые модули, иcпользуемые в веб-приложениях. Во-вторых, адаптеры соединений для различных веб-серверов на платформе `Erlang VM`. Хотя `Plug` и не является частью ядра `Elixir`, это официальный проект от той же команды.
+Если вы знакомы с `Ruby`, то можете думать о `Plug` как о комбинации `Rack` и `Sinatra`. Это набор договорённостей и спецификаций для модулей, иcпользуемых в веб-приложениях, а также адаптеры соединений для различных веб-серверов. Хотя `Plug` и не является частью ядра `Elixir`, это официальный проект от той же команды.
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: It seems that the letter `c` in the word `используем` is not Russian.

File: ru/lessons/specifics/plug.md [hunk 2]
```diff
@@ -1,54 +1,171 @@
...
+
+Мы начнем с создания минимального рабочего веб-приложения с использованием `Plug`. После этого мы познакомимся с роутерами и узнаем, как добавить `Plug` к уже существующему приложению.
 
 {% include toc.html %}
 
-## Установка
+## Перед установкой
+
+Чтобы следовать инструкциям этого урока вам понадобятся установленный Elixir версии 1.4 или выше и `mix`.
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: `вам понадобятся` → `вам, понадобятся`

File: ru/lessons/specifics/plug.md [hunk 3]
```diff
@@ -1,54 +1,171 @@
...
+```elixir
+def application do
+  [
+    extra_applications: [:logger],
+    mod: {Example, []}
+  ]
+end
+```
+
+Теперь всё готово к запуску нашего первого веб-приложение, созданного на базе `Plug`. В командной строке, выполним:
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: `строке, выполним` → `строке выполним`

File: ru/lessons/specifics/plug.md [hunk 4]
```diff
@@ -1,54 +1,171 @@
...
+
+```shell
+$ mix run --no-halt
+```
+
+Как только все скомпилируется, и выведется сообщение `[info]  Started app`, откройте в браузере `127.0.0.1:8080`. Там должно появится следующее:
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: `должно появится` → `должно появиться`

File: ru/lessons/specifics/plug.md [hunk 5]
```diff
@@ -1,54 +1,171 @@
...
+
+```
+Hello World!
+```
+
+## Использование Plug.Router
+
+Для большинства приложений, таких как веб-сайты и REST API, понадобится что-то, что будет перенаправлять запросы к определенным ресурсам на соответствующие обработчики в коде.
+Специально для этого в `Plug` существует маршрутизаор (или роутер). Как мы сейчас увидим, фреймворк типа `Sinatra` в `Elixir` не требуется, так как мы получаем его возможности вместе с `Plug`
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: `маршрутизаор` → `маршрутизатор`

File: ru/lessons/specifics/plug.md [hunk 6]
```diff
@@ -1,54 +1,171 @@
...
+def start(_type, _args) do
+  children = [
+    Plug.Adapters.Cowboy.child_spec(:http, Example.Router, [], port: 8080)
+  ]
+  Logger.info "Started application"
+  Supervisor.start_link(children, strategy: :one_for_one)
+end
+```
+
+Запустим веб-сервер (в случае, если предыдущий сервер еще работает, его можно остановить дважды нажав `Ctrl+C`).
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: `остановить дважды нажав` → `остановить, дважды нажав`

File: ru/lessons/specifics/plug.md (hunk 1 above)
Comment (martosaur) [reply ru/lessons/specifics/plug.md]: sharp eye!

**Comment by martosaur** [comment]
@nscyclone should be better now

File: ru/lessons/specifics/plug.md [hunk 7]
```diff
@@ -1,54 +1,171 @@
...
+```elixir
+def application do
+  [
+    extra_applications: [:logger],
+    mod: {Example, []}
+  ]
+end
+```
+
+Теперь всё готово к запуску нашего первого веб-приложение, созданного на базе `Plug`. В командной строке выполним:
```
Comment (nscyclone) [inline ru/lessons/specifics/plug.md]: Sorry, @martosaur, I didn't notice this one earlier:
`нашего первого веб-приложение` → `нашего первого веб-приложения`

**Comment by martosaur** [comment]
@nscyclone I should probably start using grammarly or smth :man_shrugging:

**Comment by nscyclone** [comment]
@martosaur, you're doing really great! The translation process is not that easy as it may seem on the first sight. During it there are a lot of things have to be kept in mind, so this kind of shortcomings is a normal practice. That's where the teamwork with discussion comes to help, and there is absolutely nothing bad about it.
Huge thanks once again!

**Pull request merged**